python_modules/dagster/dagster/_grpc/client.py

```diff
@@ -367,6 +367,12 @@ def streaming_external_repository(
                 "serialized_external_repository_chunk": res.serialized_external_repository_chunk,
             }
 
+    def _is_unimplemented_error(self, e: Exception):
+        return (
+            isinstance(e.__cause__, grpc.RpcError)
+            and cast(grpc.RpcError, e.__cause__).code() == grpc.StatusCode.UNIMPLEMENTED
+        )
```

Comment:
how confident are we that this is a stable detection scheme? anything in there docs confirm?

Reply:
we use it elsewhere - these are all part of the public / top-level grpc API and I think the StatusCodes have stayed stable as long as we've been using grpc

If they moved or renamed them i think a test failure would pop up quickly, i think moving UNIMPLEMENTED to some other verb is unlikely